usr/share/rear/layout/prepare/GNU/Linux/210_load_multipath.sh

```diff
@@ -1,7 +1,10 @@
 
-if grep -q '^multipath' "$LAYOUT_FILE" ; then
+# Activating multipath if BOOT_OVER_SAN variable is true
+# or if multipath device are present in LAYOUT_FILE
+if grep -q '^multipath' "$LAYOUT_FILE" || is_true "$BOOT_OVER_SAN" ; then
     Log "Activating multipath"
     modprobe dm-multipath >&2
+    [ ! -f /etc/multipath.conf ] && touch /etc/multipath.conf
```

Comment:
@schlomo 
many thanks for spotting that possible issue!

In general ReaR must never ever change the original system
(except creating ReaR's own files in /tmp/rear.XXX...).

@gdha 
what excatly do you mean with "this is on recovered system"?

As far as I see 210_load_multipath.sh
is only run during "rear recover" and then
touch /etc/multipath.conf
creates /etc/multipath.conf in the running recovery system
where ReaR can do whatever it needs
but this way it will not create /etc/multipath.conf
in the recovered system because to create
a file in the recovered system something like
touch $TARGET_FS_ROOT/etc/multipath.conf
would have to be used.

@schabrolles 
is my understanding right or do I confuse something here?